confignode/src/main/java/org/apache/iotdb/confignode/consensus/response/DataPartitionDataSet.java

```diff
@@ -97,14 +98,12 @@ public void convertToRpcDataPartitionResp(TDataPartitionResp resp) {
                             regionReplicaSets.forEach(
                                 regionReplicaSet -> {
                                   TRegionReplicaSet tRegionReplicaSet = new TRegionReplicaSet();
-
                                   // Set TRegionReplicaSet's RegionId
-                                  tRegionReplicaSet.setRegionId(
-                                      regionReplicaSet.getConsensusGroupId().getId());
-
-                                  // Set TRegionReplicaSet's GroupType
-                                  tRegionReplicaSet.setGroupType("DataRegion");
-
+                                  ByteBuffer buffer =
+                                      ByteBuffer.allocate(Byte.BYTES + Integer.BYTES);
+                                  regionReplicaSet.getConsensusGroupId().serializeImpl(buffer);
+                                  buffer.flip();
+                                  tRegionReplicaSet.setRegionId(buffer);
```

Comment:
why?

Reply:
We redefine the regionId in TRegionReplicaSet. You can see [commmon.thrift](https://y8dp9fjm8f.feishu.cn/docs/doccnwYEhSIw2a1Qo1F2KXsfLIb#zSavsx) for more details~